packages/salesforce-adapter/src/filters/extra_dependencies.ts

```diff
@@ -62,6 +62,7 @@ type Dependency = {
 type QueryDepsParams = {
   client: SalesforceClient
   elements: Element[]
+  initialChunkSize?: number
```

Comment:
```suggestion
  initialChunkSize: number
```
Shouldn't be optional at this point IMO. Pass the default when undefined.